Source/WebCore/platform/graphics/mac/controls/SliderThumbMac.mm

```diff
@@ -72,12 +64,11 @@
 void SliderThumbMac::draw(GraphicsContext& context, const FloatRoundedRect& borderRect, float deviceScaleFactor, const ControlStyle& style)
 {
     LocalDefaultSystemAppearance localAppearance(style.states.contains(ControlStyle::State::DarkAppearance), style.accentColor);
-    LocalCurrentGraphicsContext localContext(context);
```

Comment:
Is this safe to remove? We're still transforming the context below (L74).

Reply:
`GraphicsContextStateSaver` will be kept. I made `LocalCurrentGraphicsContext` be instantiated by `ControlMac::drawCellOrFocusRing()` which is called by `ControlMac::drawCell()` because the drawing `GraphicsContext` can be switched to an intermediate `ImageBuffer`.